playwright-e2e/page-objects/local-authority/birth-mother/laBirthMotherNamePage.page.ts

```diff
@@ -0,0 +1,24 @@
+import { Locator, Page } from '@playwright/test';
+
+import BasePage from '../../../pages/basePage.page';
+
+export class LABirthMotherNamePage extends BasePage {
+  readonly birthMotherFullNameHeading: Locator;
+
+  readonly birthMotherFirstNameLabel: Locator;
+  readonly birthMotherLastNameLabel: Locator;
+
+  constructor(page: Page) {
+    super(page);
+
+    this.birthMotherFullNameHeading = page.getByRole('heading', { name: 'What is the full name of the' });
+
+    this.birthMotherFirstNameLabel = page.getByLabel('First names');
+    this.birthMotherLastNameLabel = page.getByLabel('Last names');
+  }
```

Comment:
this looks like something that could be extracted to a namesPage (birth mother, birth father, possibly Other Person...), but okay if you just want to add to your to dos.  (There's a lot of crossover between those 3 sections.)

Reply:
Same for address lookup.

Reply:
And dates (address last confirmed, placement order date, DOBs)

Reply:
I 100% agree, I think this could be done in a different PR, I can make a small Jira and confirm that these fields will unlikely change and then use the Jira to cement that establishment. Also I am concerned with the size of this PR and feel like making more adjustments will create more a larger PR. I hope this information helps.

Reply:
This is to be done in a later PR, a new Jira for the backlog will be created in relations to this